controllers/apiv3/api_base_controller.py

```diff
@@ -122,9 +122,21 @@ def _validate_tba_auth_key(self):
         else:
             auth = ApiAuthAccess.get_by_id(x_tba_auth_key)
             if auth and auth.is_read_key:
-                self.auth_owner = auth.owner.id()
+                self.auth_owner = auth.owner
                 self.auth_description = auth.description
                 logging.info("Auth owner: {}, X-TBA-Auth-Key: {}".format(self.auth_owner, x_tba_auth_key))
             else:
                 self._errors = json.dumps({"Error": "X-TBA-Auth-Key is invalid. Please get an access key at http://www.thebluealliance.com/account."})
                 self.abort(400)
+
+class ApiSuggestBaseController(ApiBaseController):
+    def post(self, *args, **kw):
+        super(ApiSuggestBaseController, self).get(*args, **kw)
```

Comment:
I think I'd rather add a `post` method to `ApiBaseController` and duplicate the contents of `get`, just without the `super` call. You'd still have to call `_render` in the subclass, though. [See here](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/controllers/base_controller.py#L74)